lib/ansible/modules/network/cloudengine/ce_aaa_server.py

```diff
@@ -1697,8 +1700,6 @@ def main():
         local_user_group=dict(type='str')
     )
 
-    argument_spec.update(ce_argument_spec)
```

Comment:
Now you're also removing `provider`, which has not been deprecated and is not scheduled for removal. It might be better to simply remove `ce_top_spec` from module_utils/network/cloudengine/ce.py and then try to update everything else.

Reply:
Yes, you are right.It is my misstake.
Maybe you mean remove these [lines](https://github.com/ansible/ansible/blob/8aec05847334449ecff64de8888569ad5ce2e239/lib/ansible/module_utils/network/cloudengine/ce.py#L73)?
I also plan to fix some ingores,and to do it next ...

Reply:
Yes, these should be removed. Also, I'd suggest that you move the `type: xxx` updates to another PR, since these are things that could (and should) be backported to stable-2.9. Feature removal cannot be backported.

Reply:
Done